security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+Security is at the heart of a healthy and dependable software ecosystem and therefore also at the heart of the SSWG. This page intends to give clear and easy to follow information to users and developers of packages listed on the SSWG's package index.
+
+If you find any information to be missing, unclear, or misleading, please contact any member of the SSWG or create a post in the [server area](https://forums.swift.org/c/server/serverdev/14) of the Swift Forums.
+
+## I found a security vulnerability, what should I do?
+
+If you found a security vulnerability, first of all thank you for caring about our ecosystem. We rely on the community to keeping us all as safe and secure as we can (at least when it comes to software).
+
+The SSWG only takes a secondary role in the security process. The SSWG makes sure that listed projects do have a security process and that they follow their own process but the SSWG is typically not the first point of contact. Please don't read this as sending you away though, if at any point in the process you feel like you don't know where to report, you don't get a response, or face any other issues, please do contact the SSWG at [sswg-security-reports@forums.swift.org](mailto:sswg-security-reports@forums.swift.org) (this can only be seen by the SSWG members and the Swift core team).
+
+From packages on the SSWG's package index, it is required to list a clear point of contact on the project's main information page. Typically that is the `README.md` file in the project's GitHub/GitLab/... repository. It is also required that the project list the exact steps on how to report a vulnerability. Please follow these steps, the SSWG will be notified by the project directly. If you can't find any required information, you don't get a response from the project's maintainers, or you a are unsure that the vulnerability is treated with the appropriate care and urgency, please do contact the SSWG at [sswg-security-reports@forums.swift.org](mailto:sswg-security-reports@forums.swift.org).
+
+## For package maintainers: SSWG Security requirements
+
+Packages that are listed on the [SSWG's package index](https://swift.org/server/#projects) are required to follow the following guidelines around security.
+
+When it comes to security vulnerabilities it is key to make sure that somebody who discovers a vulnerability in your package can learn how to report very quickly. As the package author, you know best where to put important information around your software. In general it is probably the `README.md` file in the root of your repository on GitHub, GitLab, or similar. Below "main information page" will refer to the `README.md` or a similar easy-to-find resource with information about your project.
```

Comment:
I'd suggest calling out `SECURITY.md` as well since GitHub special cases it so that it is prominent when creating issues and so on (https://docs.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository#about-security-policies).

Reply:
Ok, sounds great!